Author: Daniel Leidert <dleidert@debian.org>
Description: Remove external image links.

--- a/html_docs/index.html
+++ b/html_docs/index.html
@@ -16,7 +16,7 @@
 <body>
 	<div id="content_container">
 		<h1>The chemical-mime-data project</h1>
-		<p>The source of this project can always be found at the <a href="http://sourceforge.net/projects/chemical-mime">Sourceforge.net project page</a>. <span class="sfnet"><a href="http://www.sourceforge.net"><img src="http://sflogo.sourceforge.net/sflogo.php?group_id=159685&amp;type=1" width="88" height="31" style="border: 0;" alt="SourceForge.net Logo" /></a></span></p>
+		<p>The source of this project can be found at the <a href="https://github.com/dleidert/chemical-mime">GitHub project page</a>.</p>
 
 		<ol>
 			<li><a title="The backgrounds of the chemical-mime-data package ..." href="#about">About the project</a></li>
--- a/xsl/cmd_website.xsl
+++ b/xsl/cmd_website.xsl
@@ -255,7 +255,7 @@
   <!-- * The template contains all static data written at the beginning of   -->
   <!-- * the HTML site(headline/title, about the project, ... such data).    -->
 	<h1>chemical-mime-data</h1>
-	<p>The source of <a href="index.html">this project</a> can be downloaded at the <a href="http://sourceforge.net/project/showfiles.php?group_id=159685&amp;package_id=179318">Sourceforge.net project page</a>. <span class="sfnet"><a href="http://www.sourceforge.net"><img src="http://sflogo.sourceforge.net/sflogo.php?group_id=159685&amp;type=1" width="88" height="31" style="border: 0;" alt="SourceForge.net Logo"/></a></span></p>
+	<p>The source of <a href="index.html">this project</a> can be downloaded at the <a href="https://github.com/dleidert/chemical-mime">GitHub project page</a>.</p>
 	<p>The released version is: <span class="version">&entversion;</span>.</p>
 	<p>The released Database version is: <span class="version"><xsl:value-of select="chemical-mime/@id"/></span>.</p>
 	<h2 id="toc">Table of Contents</h2>
